### PR TITLE
Improve SEV certificate chain ergonomics

### PR DESCRIPTION
**Pull request by connorkuehl**
Lays groundwork for linked issue.

Related: Related: https://github.com/enarx/enarx-keepldr/issues/17

### CODE REVIEW

File: src/util/cached_chain.rs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+/// Returns the "user-level" search path for the SEV
+/// certificate chain (`$HOME/.cache/amd-sev/chain`).
+pub fn home() -> Option<PathBuf> {
+    dirs::cache_dir().map(append_rest)
+}
+
+/// Returns the "system-level" search path for the SEV
+/// certificate chain (`/etc/cache/amd-sev/chain`).
```
Comment (haraldh) [inline src/util/cached_chain.rs]:
```console
# dnf install /etc/cache
No match for argument: /etc/cache
```

You are introducing a new subdir in `/etc`. I think you want `/var/cache`, which already exists.

**Review by haraldh — CHANGES_REQUESTED** [review]

File: src/util/cached_chain.rs (hunk 1 above)
Comment (haraldh) [reply src/util/cached_chain.rs]: Or use `/etc/amd-sev` directly.

Comment (connorkuehl) [reply src/util/cached_chain.rs]: Good catch, thanks. Switched to using `/var/cache`

**Review by haraldh — APPROVED** [review]

**Review by lkatalin — APPROVED** [review]
lgtm

**Pull request merged**